src/CMakeLists.txt

```
@@ -36,6 +36,23 @@ add_dependencies(rayleigh run_param_header)
 
 target_include_directories(rayleigh PRIVATE ${PROJECT_SOURCE_DIR}/src/Include ${CMAKE_CURRENT_BINARY_DIR})
 
+set(RAYLEIGH_CPU_OPTIMIZATIONS "none" CACHE PATH "Specify a CPU architecture to build for. Currently only 'native' or 'none' are supported.")
```

```suggestion
set(RAYLEIGH_CPU_OPTIMIZATIONS "none" CACHE STRING "Specify a CPU architecture to build for. Currently only 'native' or 'none' are supported.")
set_property(CACHE RAYLEIGH_CPU_OPTIMIZATIONS PROPERTY STRINGS "none" "native")
```

This makes it nicer to select the only two possible options in `ccmake`.